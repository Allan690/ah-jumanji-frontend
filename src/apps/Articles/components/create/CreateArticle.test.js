import React from 'react';
import Enzyme, {shallow} from 'enzyme/build';
import Adapter from "enzyme-adapter-react-16/build";

import ArticlePage from './CreateArticle';
import ArticlePageForm from './_articleform';


Enzyme.configure({ adapter: new Adapter() });

describe('ArticlePageForm', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<ArticlePageForm debug />);

    expect(component).toMatchSnapshot();
  });
});

describe('Create article page', () => {

  it('should render correctly in "debug" mode', () => {
    const component = shallow(<ArticlePage debug />);
    expect(component).toMatchSnapshot();
  });

});
