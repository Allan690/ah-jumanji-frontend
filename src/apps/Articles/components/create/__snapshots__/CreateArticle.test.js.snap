// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticlePageForm should render correctly in "debug" mode 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`Create article page should render correctly in "debug" mode 1`] = `
<div
  className="container auth-container"
>
  <div
    className="article-view col-md-10"
  >
    <div
      className="col-md-12"
    >
      <Connect(CreateArticle) />
    </div>
  </div>
</div>
`;
